internal/topology/variables/schema.go

```diff
@@ -166,6 +166,58 @@ func convertToAPIExtensionsJSONSchemaProps(schema *clusterv1.JSONSchemaProps, fl
 		props.XValidations = convertToAPIExtensionsXValidations(schema.XValidations)
 	}
 
+	if len(schema.OneOf) > 0 {
+		props.OneOf = make([]apiextensions.JSONSchemaProps, 0, len(schema.OneOf))
+		for idx, oneOf := range schema.OneOf {
+			o := oneOf
+			apiExtensionsSchema, err := convertToAPIExtensionsJSONSchemaProps(&o, fldPath.Child("oneOf").Index(idx))
+			if err != nil {
+				allErrs = append(allErrs, field.Invalid(fldPath.Child("oneOf").Index(idx), "",
```

Comment:
I think we can simply append the error without wrapping it (see Items, Properties, ... above)

Reply:
Removed.